oshop-dashboard/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
We want to avoid checking in comments. They are unnecessary most often than not. Only use them if you are doing something that is hard to read/hacky. Go through and delete each of the comments in this repo